src/wayland/output/mod.rs

```diff
@@ -205,13 +202,21 @@ impl Output {
                 current_mode: None,
                 preferred_mode: None,
                 xdg_output: None,
+                log,
             }),
             UserDataMap::default(),
         ));
 
-        let output = Output { inner: inner.clone() };
+        Output { inner }
+    }
 
-        let global = display.create_global(
+    /// Create a new output global with given name and physical properties
+    ///
+    /// The global is directly registered into the event loop, and this function
+    /// returns the global handle in case you wish to remove this global in the future.
```

Comment:
The doc should tell what happens if you invoke this again without removing the `Global<_>` first